packages/mdx/src/mdx-client/slideshow.tsx

```diff
@@ -3,26 +3,33 @@ import { EditorProps, EditorStep } from "../mini-editor"
 import { InnerCode, updateEditorStep } from "./code"
 import { Preview, PresetConfig } from "./preview"
 import { extractPreviewSteps } from "./steps"
+import { useInitialState } from "utils"
 
 export function Slideshow({
   children,
-  editorSteps,
+  className,
+  code,
   codeConfig,
+  editorSteps,
+  hasPreviewSteps,
+  // Set the initial slide index
+  initialSlideIndex = 0,
+  // Called when the slideshow state changes and returns the current state object
+  onChange: onSlideshowChange = () => {},
   presetConfig,
-  code,
-  className,
   style,
-  hasPreviewSteps,
   ...rest
 }: {
   children: React.ReactNode
-  editorSteps: EditorStep[]
+  className?: string
+  code?: EditorProps["codeConfig"]
   codeConfig: EditorProps["codeConfig"]
+  editorSteps: EditorStep[]
+  hasPreviewSteps?: boolean
+  initialSlideIndex?: number
```

Comment:
can you change the prop name to `start`? it's not the best name but it will be consistent with the same prop we have in `CH.Spotlight`